.github/workflows/paportal-rolling-instance-actions.yml

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Actions workflow for merges to main
+name: paportal-rolling-instance-actions
+
+on:
+  workflow_dispatch:    # allow for manual workflow triggering as needed
+
+  push:
+    branches: [ main ]
+
+jobs:
+  instance-actions:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest]
+      fail-fast: false
+    env:
+      RUNNER_DEBUG: 1
+      WF_USERNAME: davidjen@ppdevtools.onmicrosoft.com
+      WF_APPID: 8a7729e0-2b71-4919-a89a-c789d0a9720a
+      WF_TENANT_ID: 3041a058-5110-495a-a575-b2a5571d9eac
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        lfs: true
+
+    - name: Setup Node.js environment
+      uses: actions/setup-node@v2-beta
+      with:
+        node-version: 14.x
+        registry-url: https://npm.pkg.github.com
+
+    - name: Build and test
```

Comment:
nit: change the step title: in this case, it is only initializing the node modules so that gulp can run to fetch the paportal nupkg

Reply:
Done